_sass/components/_intro.scss

```diff
@@ -25,7 +25,7 @@
   }
 
   #content {
-    background: $primary;
+    background-image: linear-gradient(#4c6c36, #263b17);
```

Comment:
Ditto. Better to extract color definition into  /assets/css/style.scss

Reply:
added "content-singer-background", same as "btn-singer". we can remove it later